client/src/main/java/org/schlunzis/kurtama/client/service/impl/LobbyService.java

```diff
@@ -59,7 +59,11 @@ public void leaveLobby() {
 
     @EventListener
     void onUserLeftLobbyMessage(UserLeftLobbyMessage ullm) {
-        Platform.runLater(() -> lobbyUsersList.remove(ullm.leavingLobbyMember().getUsername()));
+        Platform.runLater(() -> {
+            lobbyUsersList.clear();
+            ullm.lobby().getUsers().forEach(u ->
+                    lobbyUsersList.add(u.getUsername() + (u.equals(ullm.lobby().getOwner()) ? " (Owner)" : "")));
```

Comment:
A list item factory would be cleaner.

Reply:
You mean something like changing the `lobbyUsersList` to contain Users and in the controller we set a cell factory for the `usersListView`? Comparable to the `lobbyListView` in the main menu.

Reply:
Correct